### PR TITLE
Added the Planning Scene

### PR DESCRIPTION
**Pull request by AndrewWilkinson88**
Added the Planning Scene
Deleted the SampleScene
Added Assets for UI

### CODE REVIEW

File: Assets/DragDrop.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public enum DropType
+{ 
+    None,
```
Comment (kkohler-luckypoint) [inline Assets/DragDrop.cs:8]: probably doesn't matter -- but I used _Arm_ _Body_ _Feet_ _Head_

File: Assets/DragDrop.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    void Start()
+    {
+        canvas = PlanningSceneManager.Instance.gameCanvas;
+        rectTransform = GetComponent<RectTransform>();
+        canvasGroup = GetComponent<CanvasGroup>();
+    }
+
+    public void OnPointerDown(PointerEventData eventData)
+    {
+        //Not Implemented
```
Comment (williamhammond) [inline Assets/DragDrop.cs]: nit pick but you could do `throw new NotImplementedException();` or `Debug.Log("OnPointerDown ran " + eventDate)`

File: Assets/DragDrop.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    public void OnBeginDrag(PointerEventData eventData)
+    {
+        transform.parent.GetComponent<DropSlot>().ClearSlot();
+        lastPosition = transform.parent;
+        transform.SetParent(canvas.transform, true);
+        transform.SetAsLastSibling();
+        canvasGroup.alpha = 0.6f;
```
Comment (williamhammond) [inline Assets/DragDrop.cs:38]: Are these things that might be useful to have available in the editor to tweak?

File: Assets/PlanningSceneManager.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+            {
+                Debug.LogError("Planning Scene Manager is NULL!");
+            }
+            return _instance;
+        }
+    }
+
+    private void Awake()
+    {
+        _instance = this;
```
Comment (williamhammond) [inline Assets/PlanningSceneManager.cs:24]: I think this might happen automatically? I'm not really sure though

**Review by kkohler-luckypoint — APPROVED** [review]
Looks good!

File: Assets/PlanningSceneManager.cs (hunk 4 above)
Comment (AndrewWilkinson88) [reply Assets/PlanningSceneManager.cs:24]: sadly it does not :( this is how you can sort of fake Singletons in unity

File: Assets/DragDrop.cs (hunk 3 above)
Comment (AndrewWilkinson88) [reply Assets/DragDrop.cs:38]: these ones in particular I don't wanna expose to the editor!

File: Assets/DragDrop.cs (hunk 1 above)
Comment (AndrewWilkinson88) [reply Assets/DragDrop.cs:8]: Updated

File: Assets/DragDrop.cs (hunk 2 above)
Comment (AndrewWilkinson88) [reply Assets/DragDrop.cs]: Good Point, if I'm not using it I should just remove the interface

**Pull request merged**